src/main/resources/db/migration/V208__add_new_column_epim_id.sql

```diff
@@ -0,0 +1,3 @@
+-- FACT-1903
+-- Add new column epim_id to court address table
+ALTER TABLE search_courtaddress ADD COLUMN IF NOT EXISTS epim_id VARCHAR(255);
```

Comment:
Length needs to be 30

Reply:
Probably need to validate the length as well

Reply:
changed